packages/fuels/tests/harness.rs

```diff
@@ -3671,6 +3632,7 @@ async fn test_rust_result_can_be_decoded() -> Result<(), Box<dyn std::error::Err
     Ok(())
 }
 
+/*
```

Comment:
Isn't this a bit early to open the comment block? I think you caught a bit of the tests for `Result`s usage in sway contracts?